snekbox/nsjail.py

```diff
@@ -102,19 +98,27 @@ def _create_dynamic_cgroups(self) -> str:
         pids.mkdir(parents=True, exist_ok=True)
         mem.mkdir(parents=True, exist_ok=True)
```

Comment:
This also needs to be adjusted, since v2 doesn't have subdirectories for each controller. NsJail doesn't even use `cgroup_pids_mount` and `cgroup_mem_mount` for v2. Instead, it's simply https://github.com/google/nsjail/blob/bf93e8a25d1b521797a319bc1f6182e894597224/cgroup2.cc#L42-L44